### PR TITLE
[6.3] FIX  API UI test_errata CDN repos

### PR DESCRIPTION
**Pull request by ldjebran**
```console
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test -v tests/foreman/api/test_errata.py::ErrataTestCase::test_positive_get_applicable_for_host
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.1.3, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.2, cov-2.4.0
collected 1 item 
2017-08-18 12:18:00 - conftest - DEBUG - Found WONTFIX in decorated tests ['1147100', '1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1311113', '1199150', '1204686', '1221971', '1103157', '1230902', '1230865', '1214312', '1079482']

2017-08-18 12:18:00 - conftest - DEBUG - Collected 1 test cases


tests/foreman/api/test_errata.py::ErrataTestCase::test_positive_get_applicable_for_host PASSED

================================================== 0 tests deselected ==================================================
============================================= 1 passed in 1108.28 seconds ==============================================
```
```console
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test tests/foreman/ui/test_errata.py::ErrataTestCase -v -k "test_positive_show_count_on_chost_details_page or test_positive_show_count_on_chost_page"
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.1.3, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.2, cov-2.4.0
collected 17 items 
2017-08-18 13:03:23 - conftest - DEBUG - Found WONTFIX in decorated tests ['1147100', '1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1311113', '1199150', '1204686', '1221971', '1103157', '1230902', '1230865', '1214312', '1079482']

2017-08-18 13:03:23 - conftest - DEBUG - Collected 17 test cases


tests/foreman/ui/test_errata.py::ErrataTestCase::test_positive_show_count_on_chost_details_page PASSED
tests/foreman/ui/test_errata.py::ErrataTestCase::test_positive_show_count_on_chost_page PASSED

================================================= 15 tests deselected ==================================================
====================================== 2 passed, 15 deselected in 2521.69 seconds ======================================
```

### CODE REVIEW

File: tests/foreman/ui/test_errata.py [hunk 1]
```diff
@@ -728,17 +728,14 @@ def test_positive_show_count_on_chost_page(self):
                 result = self.contenthost.fetch_errata_counts(client.hostname)
                 for errata in ('security', 'bug_fix', 'enhancement'):
                     self.assertEqual(result[errata]['value'], 0)
-                    self.assertEqual(result[errata]['color'], 'black')
```
Comment (abalakh) [inline tests/foreman/ui/test_errata.py]: Take a look at expected results, these colors should be verified according to docstring, why are you removing the checks?

Comment (ldjebran) [inline tests/foreman/ui/test_errata.py]: because they do not exist in the class attribute, and they cannot be verified

Comment (ldjebran) [inline tests/foreman/ui/test_errata.py]: it make no sense to restore them grammatically from this string (the string it self compare to the numbers)
```
<i class="fa fa-warning inline-icon" ng-class="{black: host.content_facet_attributes.errata_counts.security === 0, red: host.content_facet_attributes.errata_counts.security > 0}" title="Security"></i>
```

Comment (abalakh) [inline tests/foreman/ui/test_errata.py]: Then i guess it should be dropped from `def fetch_errata_counts()` as well.

**Comment by ldjebran** [comment]
put do not merge to make changes

**Comment by ldjebran** [comment]
@abalakh comments addressed opened a bug for that black color https://bugzilla.redhat.com/show_bug.cgi?id=1484044
```console
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test -v tests/foreman/ui/test_errata.py::ErrataTestCase -v -k "test_positive_show_count_on_chost_details_page or test_positive_show_count_on_chost_page"
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.1.3, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.2, cov-2.4.0
collected 17 items 
2017-08-22 18:10:32 - conftest - DEBUG - Found WONTFIX in decorated tests ['1147100', '1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1311113', '1199150', '1204686', '1221971', '1103157', '1230902', '1230865', '1214312', '1079482']

2017-08-22 18:10:32 - conftest - DEBUG - Collected 17 test cases


tests/foreman/ui/test_errata.py::ErrataTestCase::test_positive_show_count_on_chost_details_page PASSED
tests/foreman/ui/test_errata.py::ErrataTestCase::test_positive_show_count_on_chost_page PASSED

================================================= 15 tests deselected ==================================================
====================================== 2 passed, 15 deselected in 1478.61 seconds ======================================
```

**Pull request merged**